src/GraphQL/Types/GraphType.cs

```diff
@@ -51,6 +51,9 @@ private string GetDefaultName()
         {
             var type = GetType();
```

Comment:
Should it be
```c#
return type.GraphQLName();
```
after this line, and the rest of the method removed?

Reply:
No, because the legacy code worked different than the legacy version of `.GraphQLName()`.